plugins/github-deployments/src/components/GithubDeploymentsCard.tsx

```diff
@@ -28,21 +28,27 @@ import {
   isGithubDeploymentsAvailable,
 } from '../Router';
 import { GithubDeploymentsTable } from './GithubDeploymentsTable/GithubDeploymentsTable';
+import {
+  LOCATION_ANNOTATION,
+  SOURCE_LOCATION_ANNOTATION,
+} from '@backstage/catalog-model';
 
 const GithubDeploymentsComponent = ({
   projectSlug,
   last,
   columns,
+  locations,
 }: {
   projectSlug: string;
   last: number;
+  locations: string[];
   columns: TableColumn<GithubDeployment>[];
 }) => {
   const api = useApi(githubDeploymentsApiRef);
   const [owner, repo] = projectSlug.split('/');
 
   const { loading, value, error, retry: reload } = useAsyncRetry(
-    async () => await api.listDeployments({ owner, repo, last }),
+    async () => await api.listDeployments({ owner, repo, last }, { locations }),
```

Comment:
Would it be better with `listDeployments({ host, owner, repo, last })` where the host is an optional parameter? That's a good lookup key for the scmintegrations api